tests/test_suite.py

```diff
@@ -206,8 +209,8 @@ def generateWorldsList(groupName):
                     not ('GITHUB_ACTIONS' in os.environ and (
                         filename.endswith('speaker.wbt') or
                         filename.endswith('local_proto_with_texture.wbt') or
-                        (filename.endswith('robot_window_html.wbt') and is_ubuntu_22_04) or
-                        (filename.endswith('supervisor_start_stop_movie.wbt') and is_ubuntu_22_04) or
+                        (filename.endswith('robot_window_html.wbt') and (is_ubuntu_22_04 or is_ubuntu_24_04)) or
```

Comment:
This added condition means `supervisor_start_stop_movie.wbt` and `robot_window_html.wbt` will not be tested at all on Linux. We should check if we can test it at least for one version of Ubuntu